live-examples/html-examples/forms/meta.json

```diff
@@ -16,6 +16,14 @@
             "title": "HTML Demo: &lt;fieldset&gt;",
             "type": "tabbed"
         },
+        "form": {
+            "baseTmpl": "tmpl/live-tabbed-tmpl.html",
+            "exampleCode": "live-examples/html-examples/forms/form.html",
+            "cssExampleSrc": "live-examples/html-examples/forms/css/form.css",
+            "fileName": "form.html",
+            "title": "HTML Demo: <form>",
```

Comment:
@chrisdavidmills Ok phew, so that took a while :) This is the problem. This branch does not yet have the auto escaping in BoB so, the `form` is rendered and whenever you click on the tabs, it submits.

So, simple escape the opening `<` and all will be good with the world again :) Thanks Chris!